assets/pages/sponsor/partial.jinja

```diff
@@ -0,0 +1,22 @@
+<div class="sponsor-type">
+  <div class="row">
+    <div class="col m3">
+      {{sponsor.logs.img}}
```

Comment:
Possible typo in assets/page/sponsor/partial.jinja.

Could sponsor.logs.img be sponsor.logos.img or something else? (Or should it be here? Seems duplicate with the p.logo section.)